src/components/frame/Extensions/Header/Header.tsx

```diff
@@ -74,46 +101,92 @@ const Header = () => {
                     </div>
                   )}
                   {isMobile && (
-                    <ColonyDropdownMobile
-                      isOpen={isOpen}
-                      userLoading={userLoading}
+                    <div
+                      ref={setTooltipRef}
+                      {...getTooltipProps({
+                        className: clsx(`flex justify-start z-[9999]`),
+                      })}
                     >
-                      {!!watchlist.length && (
-                        <ColoniesDropdown
-                          watchlist={[...watchlist].sort(sortByDate)}
-                          isMobile={isMobile}
-                        />
-                      )}
-                    </ColonyDropdownMobile>
+                      <ColonyDropdownMobile
+                        isOpen={visible}
+                        userLoading={userLoading}
+                      >
+                        <div className={styles.mobileButtons}>
+                          {nativeToken && <Token nativeToken={nativeToken} />}
+                          <Button mode="tertiaryOutline" isFullRounded>
+                            <div className="flex items-center gap-3">
+                              <UserAvatar
+                                userName={
+                                  profile?.displayName || user?.name || ''
+                                }
+                                size="xxs"
+                                user={user}
+                              />
+                              <MemberReputation
+                                userReputation={userReputation}
+                                totalReputation={totalReputation}
+                                hideOnMobile={false}
+                              />
+                            </div>
+                          </Button>
+                          <Button mode="tertiaryOutline" isFullRounded>
+                            <Icon
+                              name="list"
+                              appearance={{ size: 'extraTiny' }}
+                            />
+                            {!isMobile && (
+                              <p className="text-sm font-inter font-medium ml-1">
```

Comment:
Do you need these styles here? `text-sm font-inter font-medium`
I think they are used in the Button component directly

Reply:
Removed